src/backend/sgx/mod.rs

```diff
@@ -20,11 +21,15 @@ use std::convert::TryInto;
 use std::path::Path;
 use std::sync::{Arc, RwLock};
 
+mod attestation;
 mod data;
 mod shim;
 
 const SHIM: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/bin/shim-sgx"));
 
+// See https://github.com/enarx/enarx-keepldr/issues/31
+const SYS_GETATT: usize = 0xEA01;
```

Comment:
Note: In a followup patch I will collect all these syscall consts and put them in a central place.